Makefile

```diff
@@ -1,2 +1,26 @@
 test:
 	poetry run pytest
+
+lint:
+	pre-commit run --all-files
+
+makemigrations:
+	poetry run python manage.py makemigrations
+
+recreate-first-migration:
+	poetry run python manage.py migrate api zero && \
+	poetry run python manage.py flush --no-input && \
+	mv ./api/migrations/0002_data_migration.py ./api/migrations/0002_data_migration.py.bak && \
+	rm -rf ./api/migrations/0001_initial.py && \
+	poetry run python manage.py makemigrations && \
+	mv ./api/migrations/0002_data_migration.py.bak ./api/migrations/0002_data_migration.py && \
+	poetry run python manage.py migrate
```

Comment:
```suggestion
	poetry run python manage.py migrate && \
	git add ./api/migrations/0001.initial.py
```

Reply:
If we delete a file and then re-create it, I think we have to stage it again.

Reply:
Are you sure about it? I've tested it, and it worked without `git add`

Reply:
> Are you sure about it? I've tested it, and it worked without `git add`

Hm... first of all, it fails completely at `makemigrations` stage:

```
  File "/home/mitya/PycharmProjects/django-webapps/api/urls.py", line 4, in <module>
    from api.views import (
  File "/home/mitya/PycharmProjects/django-webapps/api/views/__init__.py", line 3, in <module>
    from api.views.enrollment_test import EnrollmentTestResultViewSet, EnrollmentTestViewSet
  File "/home/mitya/PycharmProjects/django-webapps/api/views/enrollment_test.py", line 17, in <module>
    class EnrollmentTestResultViewSet(CreateModelMixin, GenericViewSet[EnrollmentTestResult]):
TypeError: 'type' object is not subscriptable
```

Reply:
Fix your settings.py with actual monkeypatch from settings_sample

Reply:
OK great. 

Well PyCharm does detect the file, but if you type `git status` in CLI you get:

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   api/migrations/0001_initial.py
```
and `git commit` will tell you that you have nothing staged.

Reply:
It will be same if you manually edit file - you will need add it to commit. 
I don't think we need to mix multiple things here, this command is about recreating migration, not about adding anything to git :)

Reply:
OK that makes sense